<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLScreenShotDialog</class>
 <widget class="QDialog" name="qMRMLScreenShotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Annotation Screenshot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Thumbnail</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="topRow" stretch="0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="fullLayoutRadio">
            <property name="text">
             <string>Full layout</string>
            </property>
            <property name="icon">
             <iconset resource="../qMRMLWidgets.qrc">
              <normaloff>:/Icons/LayoutFourUpView.png</normaloff>:/Icons/LayoutFourUpView.png</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="threeDViewRadio">
            <property name="text">
             <string>3D View</string>
            </property>
            <property name="icon">
             <iconset resource="../qMRMLWidgets.qrc">
              <normaloff>:/Icons/LayoutOneUp3DView.png</normaloff>:/Icons/LayoutOneUp3DView.png</iconset>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="redSliceViewRadio">
            <property name="text">
             <string>Red Slice View</string>
            </property>
            <property name="icon">
             <iconset resource="../qMRMLWidgets.qrc">
              <normaloff>:/Icons/LayoutOneUpRedSliceView.png</normaloff>:/Icons/LayoutOneUpRedSliceView.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="yellowSliceViewRadio">
            <property name="text">
             <string>Yellow Slice View</string>
            </property>
            <property name="icon">
             <iconset resource="../qMRMLWidgets.qrc">
              <normaloff>:/Icons/LayoutOneUpYellowSliceView.png</normaloff>:/Icons/LayoutOneUpYellowSliceView.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="greenSliceViewRadio">
            <property name="text">
             <string>Green Slice View</string>
            </property>
            <property name="icon">
             <iconset resource="../qMRMLWidgets.qrc">
              <normaloff>:/Icons/LayoutOneUpGreenSliceView.png</normaloff>:/Icons/LayoutOneUpGreenSliceView.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ctkThumbnailLabel" name="ScreenshotWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="textPosition">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="transformationMode">
           <enum>Qt::SmoothTransformation</enum>
          </property>
          <property name="selectedColor">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="scaleFactorLayout">
        <item>
         <widget class="QLabel" name="scaleFactorLabel">
          <property name="text">
           <string>Scale factor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ctkDoubleSpinBox" name="scaleFactorSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Adjust the Magnification factor.</string>
          </property>
          <property name="suffix">
           <string>x</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="saveAsButtonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsButton">
          <property name="text">
           <string>Save As...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string>Save snapshot via File Save. Edit in Annotations module.</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="descriptionTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkThumbnailLabel</class>
   <extends>QWidget</extends>
   <header>ctkThumbnailLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullLayoutRadio</sender>
   <signal>clicked()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threeDViewRadio</sender>
   <signal>clicked()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSliceViewRadio</sender>
   <signal>clicked()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yellowSliceViewRadio</sender>
   <signal>clicked()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSliceViewRadio</sender>
   <signal>clicked()</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleFactorSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>qMRMLScreenShotDialog</receiver>
   <slot>grabScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>grabScreenShot()</slot>
 </slots>
</ui>
